app/javascript/lib/metrics_table_sort.js

```diff
@@ -19,16 +29,51 @@ function setupOrgLevelCounts() {
         });
 
         // Set viewable value and sortable data-js-count value for org based on accumulated value.
-        if (attention_count > 0) {
-            $(this).find('.attention-needed-breach').first().text(attention_count).attr('data-js-count', attention_count);
-        }
-        if (communication_count > 0) {
-            $(this).find('.communication-breach').first().text(communication_count).attr('data-js-count', communication_count);
-        }
-        if (interaction_count > 0) {
-            $(this).find('.interaction-breach').first().text(interaction_count).attr('data-js-count', interaction_count);
-        }
+        $(this).find('.attention-needed-breach').first().text(attention_count).attr('data-js-count', attention_count);
```

Comment:
This is crazy repetitive and can be cleaned up, but just wanted to get it working.